common/Cargo.toml

```diff
@@ -13,6 +13,7 @@ bitflags = { workspace = true }
 
 [dev-dependencies]
 static_assertions = "1.1.0"
+clack-plugin = { workspace = true }
```

Comment:
Which tests require the clack-plugin dependency on clack-common? I cannot reproduce this on my end. :/